tests/addRemove.test.js

```diff
@@ -0,0 +1,80 @@
+/*
+* @jest-environment jsdom
+*/
+
+/* eslint-disable global-require */
+/* eslint-disable import/first */
```

Comment:
- [ ] [Optional] It's not a good practice to ignore the linters, please fix them instead of ignoring them so your code follows specific rules every time

Reply:
Thank you @Ismailco 

I have updated with the ignore removed.